packages/components/src/components/SquiggleViewer/getWidget.tsx

```diff
@@ -1,16 +1,19 @@
-import React, { ReactNode } from "react";
```

Comment:
Maybe rename to ``getValueWidget``? Also, we might want to rename all of the "Value" names in filenames to "SqValue" or something more specific. Could do in a future story/refactor though.